cmd/launcher/launcher.go

```diff
@@ -116,6 +117,7 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 			"err", err,
 		)
 	}
+	startupSpan.AddEvent("dns_lookup_complete")
```

Comment:
just double checking you don't want `completed` here instead to match the others

Reply:
Ah whoops, thank you! Updated